application/models/Question.php

```diff
@@ -207,6 +207,9 @@ public function rules()
                 $aRules[] = array('other', 'compare', 'compareValue' => 'Y', 'operator' => '!=', 'message' => sprintf(gT("'%s' can not be used if the 'Other' option for this question is activated."), 'other'));
             }
         }
+        if ($this->survey->isActive) {
+            $aRules = array_merge($aRules, $this->rulesForActiveSurvey());
```

Comment:
No : we merge why `over-ride` ?

Reply:
PS : i start with `$aRules + $this->rulesForActiveSurvey()` (then no new rules … silly boy)

Reply:
OK, nevermind. I set this comment 5 days ago and somehow was kept on "pending".
Now looks OK.

> I would check if survey is empty, just as for sanity, just in case to avoid unexpected situations and dumps.

This holds. Not sure if it is worth to address it.

Reply:
> This holds. Not sure if it is worth to address it.

Did question can be called in this situation ?  Survey model is not used before ?

I'm sure we don't have it in current call, but …

Reply:
one other place

https://github.com/LimeSurvey/LimeSurvey/blob/4920978b1a27514b0dc3404a15a4bec6a79f2980/application/models/Question.php#L720 (but with survey::model call before)

Reply:
My opinion : we must throw an error BEFORE try to save a question without related survey (and related group) …